pkg/openshift/rosa/rosa.go

```diff
@@ -234,6 +240,7 @@ func New(ctx context.Context, token string, clientID string, clientSecret string
 	if err != nil {
 		return nil, &providerError{err: fmt.Errorf("aws credential set and validation failed: %v", err)}
 	}
+	isFedRamp := fedramp.IsGovRegion(awsCredentials.Region)
```

Comment:
I like this better, but instead of pulling in this dep for only this, can we just do a `string.Contains(region, "gov")`? It looks like that is standard from AWS